packages/yew-macro/tests/html_macro/generic-component-fail.rs

```diff
@@ -1,54 +1,54 @@
-use std::marker::PhantomData;
-use yew::prelude::*;
-
-pub struct Generic<T> {
-    marker: PhantomData<T>,
-}
-
-impl<T> Component for Generic<T>
-where
-    T: 'static,
-{
-    type Message = ();
-    type Properties = ();
-
-    fn create(_ctx: &Context<Self>) -> Self {
-        unimplemented!()
-    }
-    fn view(&self, _ctx: &Context<Self>) -> Html {
-        unimplemented!()
-    }
-}
-
-pub struct Generic2<T1, T2> {
-    marker: PhantomData<(T1, T2)>,
-}
-
-impl<T1, T2> Component for Generic2<T1, T2>
-where
-    T1: 'static,
-    T2: 'static,
-{
-    type Message = ();
-    type Properties = ();
-
-    fn create(_ctx: &Context<Self>) -> Self {
-        unimplemented!()
-    }
-    fn view(&self, _ctx: &Context<Self>) -> Html {
-        unimplemented!()
-    }}
-
-fn compile_fail() {
-    #[allow(unused_imports)]
-    use std::path::Path;
-
-    html! { <Generic<String>> };
-    html! { <Generic<String>></Generic> };
-    html! { <Generic<String>></Generic<Vec<String>>> };
-
-    html! { <Generic<String>></Generic<Path>> };
-    html! { <Generic<String>></> };
-}
-
-fn main() {}
+use std::marker::PhantomData;
```

Comment:
I don't know why this happened. I also saw this after my commit. The same holds for the other file you also mentioned. I do not know what I should do here, because I can't really revert anything as nothing has changed...

Reply:
After some googling, I think I know why. I originally started working on my desktop and naively copied over some files to my laptop to finish this PR. My desktop uses windows and therefore maybe uses different line feed endings (the \r\n vs \n). 

I am new to making contributions, do you know how I can rectify this? Should I make another commit where I take a copy from the original files and replace them?